src/main/java/io/cdap/plugin/gcp/publisher/source/PubSubRDDIterator.java

```diff
@@ -126,7 +132,8 @@ private SubscriberStub buildSubscriberClient() throws IOException {
     builder.setTransportChannelProvider(
       SubscriberStubSettings.defaultGrpcTransportProviderBuilder()
         .setMaxInboundMessageSize(MAX_MESSAGE_SIZE).build());
-    builder.getSubscriptionSettings().setRetrySettings(PubSubSubscriberUtil.getRetrySettings());
+    builder.getSubscriptionSettings().setRetrySettings(PubSubSubscriberUtil.getRetrySettings())
```

Comment:
Does this add to the default codes in the client or does it replace ?

Reply:
It replaces the codes. I have added the default codes in the comment.